$OpenBSD: patch-ipc_chromium_src_base_atomicops_h,v 1.4 2012/05/09 17:16:01 landry Exp $
../../ipc/chromium/src/base/singleton.h:118: error: invalid conversion from 'base::subtle::AtomicWord*' to 'const volatile base::subtle::Atomic64*'
--- ipc/chromium/src/base/atomicops.h.orig	Wed Dec  7 06:27:43 2011
+++ ipc/chromium/src/base/atomicops.h	Sat Dec 10 17:33:13 2011
@@ -40,7 +40,7 @@ namespace subtle {
 #endif
 typedef __w64 int32 Atomic32;
 #ifdef ARCH_CPU_64_BITS
-typedef int64 Atomic64;
+typedef intptr_t Atomic64;
 #endif
 
 // Use AtomicWord for a machine-sized pointer.  It will use the Atomic32 or
